src/utils.jsx

```diff
@@ -1,6 +1,17 @@
 import { DATE_RANGE_MIN, SLUG_SEPARATOR } from './constants'
 import moment from 'moment'
 
+/**
+* Breaks up '123' to '1 2 3' to help screen readers read digits individually
+* https://thatdevgirl.com/blog/accessibility-phone-number-formatting
+*
+* @param {string} s the string of digits
+* @returns {string} an expanded string of digits
+*/
+export function ariaReadoutNumbers( s ) {
```

Comment:
slick 👍

Reply:
I know this is tested via the snapshots, but can we get a test?

Reply:
sure!